### ISSUE TITLE
I'm not certain the /XD option is fully explained?

### ISSUE THREAD

**Issue by Floppity** [MicrosoftDocs/windowsserverdocs#6312]
I did this (in a .cmd file):
`set EXCLUDEDIR=*x64 *Previews.lrdata
robocopy %SRC%\%NAME% %DEST%\%NAME% /e /xo /r:3 /z /dcopy:DAT /purge /XD %EXCLUDEDIR% /log+:%CLOG% /tee`

This is an attempt at a cloud upload excluding build output and Lightroom previews. It does, actually, appear to work - directories of both names are excluded. The document page doesn't mention multiple folder options for /XD.

And, when Robocopy echoes it's command, it comes out like this:

`   ROBOCOPY     ::     Robust File Copy for Windows                              
  Started : 02 March 2022 13:52:31
   Source : \\SILVER_NAS1\DocumentsNAS1\SharedToCloud\
    Files : *.*
 Exc Dirs : *x64
  Options : *.* /TEE /S /E /DCOPY:DAT /COPY:DAT /PURGE /Z /XO /R:3 /W:30 `

(As printed by Robocopy that has a few more line separations, lost when pasted here. It's years since I had CRLF issues... takes me back! Anyway.)

Which is to say, it has reported the excluded directories incompletely.

This is on an up to date W10 as of 2nd March 2022.

---
#### Document Details

⚠ *Do not edit this section. It is required for docs.microsoft.com ➟ GitHub issue linking.*

* ID: bf2281d4-1470-0658-ec5a-687fe2cd451c
* Version Independent ID: 2ce2be3e-e3a2-790c-3bff-90efe7318450
* Content: [robocopy](https://docs.microsoft.com/en-us/windows-server/administration/windows-commands/robocopy)
* Content Source: [WindowsServerDocs/administration/windows-commands/robocopy.md](https://github.com/MicrosoftDocs/windowsserverdocs/blob/main/WindowsServerDocs/administration/windows-commands/robocopy.md)
* Product: **windows-server**
* Technology: **windows-commands**
* GitHub Login: @JasonGerend
* Microsoft Alias: **jgerend**

**Comment by Xelu86** [comment]
#assign:xelu86

**Comment by Xelu86** [comment]
Hey @Floppity,

Thanks again for contacting Microsoft. We value your feedback.

Looking at the entire documentation itself, it appears to be assumed that the user(s) already have knowledge of how windows syntax works, such as continuing to add directories after the /XD parameter with quotes surrounding directory names with spaces.

The description does say "directories" to show that multiple directories are selectable:<html>
<body>


/xd \<directory>[ ...] | Excludes directories that match the specified names and paths.
-- | --



</body>
</html>

I ran my own robocopy /xd parameter to test:

![image](https://user-images.githubusercontent.com/103963494/183489159-f5410802-5c7e-4568-a113-ccd9df170657.png)

I later added all the parameters in your example, to replicate any errors, and on my end, it did report correctly (shows 5 skipped because I ran it 2x):

![image](https://user-images.githubusercontent.com/103963494/183496054-4594013a-8f3d-46c0-92b0-3ac86d9eeff7.png)

I do know that robocopy supports long file names by default so it can't be a character limit. I am unable to determine what is the cause of the error you've experienced. 

Hopefully your main question was answered.
